lib/package_config_types.dart

```diff
@@ -3,7 +3,13 @@
 // BSD-style license that can be found in the LICENSE file.
 
 /// A package configuration is a way to assign file paths to package URIs,
-/// and vice-versa,
+/// and vice-versa.
+
```

Comment:
I think this line needs a `///` too, otherwise I'm not sure the documentation above is associated with the library.

Reply:
Added